frontend/packages/dev-console/src/utils/useAddToProjectAccess.ts

```diff
@@ -36,6 +45,10 @@ export const useAddToProjectAccess = (activeNamespace: string): string[] => {
   const routeAccess = useAccessReview(resourceAttributes(RouteModel, activeNamespace));
   const serviceAccess = useAccessReview(resourceAttributes(ServiceModel, activeNamespace));
 
+  const serviceBindingEnabled = useSelector((state: RootState) =>
+    state.FLAGS.get(ALLOW_SERVICE_BINDING),
+  );
```

Comment:
I'm surprised we don't have a `useFeatureFlag` hook. We should create one inside `features.ts`